src/firewall-config

```diff
@@ -279,11 +279,11 @@ class FirewallConfig(object):
                                               GObject.TYPE_STRING, # table
                                               GObject.TYPE_STRING) # chain
         self.directChainView.append_column(
-            Gtk.TreeViewColumn("ipv", Gtk.CellRendererText(), text=0))
+            Gtk.TreeViewColumn(_("ipv"), Gtk.CellRendererText(), text=0))
```

Comment:
I don't think there in any way to translate "ipv" as it's not really a word, but more of a firewalld created term. What do you think?